aspnetcore/blazor/call-javascript-from-dotnet.md

```diff
@@ -148,6 +148,10 @@ The placeholder `{APP ASSEMBLY}` is the app's app assembly name (for example, `B
 
 JavaScript functions that return [void(0)/void 0](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Operators/void) or [undefined](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/undefined) are called with <xref:Microsoft.JSInterop.JSRuntimeExtensions.InvokeVoidAsync%2A?displayProperty=nameWithType>.
 
+## Call an asynchronous JavaScript function
```

Comment:
Instead of a section, couldn't this paragraph appear at Line 23? ... right where `IJSRuntime.InvokeAsync` is explained.

Reply:
Yeah, that sounds fine! I wasn't sure if it would be too hidden if it was up there.